tosan-httpserver-spring-boot-starter/src/test/java/com/tosan/http/server/starter/util/HttpLogUtilUTest.java

```diff
@@ -48,8 +47,6 @@ public void setup() throws IOException {
         when(response.getStatus()).thenReturn(responseStatus);
```

Comment:
this class is not responsible for creating log content but only to provide information for log provider...so why log provider is not mocked? and no specific test for logProvider